Makefile

```diff
@@ -2,8 +2,8 @@
 .PHONY : test
 
 update-dependencies:
-	docker pull jwilder/docker-gen:latest
-	docker pull nginx:latest
+	docker pull jwilder/docker-gen:0.7.0
+	docker pull nginx:1.9.12
```

Comment:
I have changed these to be hard dependencies to match what nginx-proxy currently uses and supports. See also [this note](https://github.com/jwilder/nginx-proxy/pull/396/commits/9b8fa06bb09e6d4086dd43ea5ea03c27c8df50bd#r57126623).